packages/itwinui-css/src/dialog/dialog.scss

```diff
@@ -181,33 +140,12 @@
   padding: 0 $iui-m;
   overflow-y: auto;
   overflow-y: overlay;
-
-  @at-root .iui-dialog-draggable & {
-    margin: 0;
-  }
 }
 
 @mixin iui-dialog-button-bar {
   margin-top: $iui-baseline;
   display: flex;
   align-items: center;
   justify-content: flex-end;
-
-  @at-root .iui-dialog-draggable & {
-    padding: 0 $iui-m $iui-baseline $iui-m;
-  }
-
-  // #region IE support
-  > .iui-button:not(:last-child) {
-    margin-right: $iui-s;
-  }
-
-  @supports (column-gap: $iui-s) {
-    column-gap: $iui-s;
-
-    > .iui-button:not(:last-child) {
-      margin-right: 0;
-    }
-  }
-  // #endregion IE support
+  gap: $iui-s;
```

Comment:
Same as with CSS varaibles, I think we still need to support IE and this will be removed with Adam's PR.